databridge.component/src/test/java/org/eclipse/digitaltwin/basyx/databridge/executable/integration/ITTestDataBridgeAASPollingConsumer.java

```diff
@@ -50,9 +47,5 @@ protected MqttClient getMqttClient() throws MqttException {
 		
 		return new MqttClient("tcp://" + BROKER_HOST+ ":1883", publisherId, new MemoryPersistence());
 	}
-	
-	@Override
-	protected IAASAggregator getAASAggregatorProxy() {
-		return new AASAggregatorProxy("http://" + HOST + ":4001");
-	}
+
```

Comment:
Remove line

Reply:
Trailing empty line at eof is on purpose because 
1. POSIX files should end with a line break
2. git diff uses the last empty line as reference. Therefore diff sometimes struggles with files that don't end with an line break